.github/workflows/ci.yml

```diff
@@ -0,0 +1,267 @@
+name: mattermost-build
+on:
+  push
+jobs:
+  check-mocks:
+    name: Check mocks
+    runs-on: ubuntu-latest-8-cores
```

Comment:
I think we can revert back to the `ubuntu-22.04` for the normal checks . We can keep the 8 cores for the integration tests